src/main/java/com/apuope/apuope_re/services/JWTService.java

```diff
@@ -0,0 +1,69 @@
+package com.apuope.apuope_re.services;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.SecretKey;
+import java.security.NoSuchAlgorithmException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JWTService {
+    private final SecretKey secretKey;
+    public JWTService() throws NoSuchAlgorithmException {
+        secretKey = Keys.secretKeyFor(SignatureAlgorithm.HS384);
+    }
+
+    public String generateToken(String email) {
+        Map<String, Object> claims = new HashMap<>();
+        return Jwts.builder()
+                .claims()
+                .add(claims)
+                .subject(email)
+                .issuedAt(new Date(System.currentTimeMillis()))
+                .expiration(new Date(System.currentTimeMillis()+ 1000 * 60 * 60 * 24))
```

Comment:
You could create global variable for the expiration time

Reply:
yes I could, if that make the code readability easier for everyone, I just thought is neat to do it like that.